### DISCUSSION TITLE
How do I filter on an event title?

### DISCUSSION THREAD

**Discussion by the-joey-o** [MMRIZE/MMM-CalendarExt3#167] [Q&A]
I'm trying to filter out a recurring event from my Google Calendar. I've tried the following and several variations with no luck:

`eventFilter: (ev) => {
  if (ev.title.includes('Weight Loss Phase')) return false
  return true
}`

The above displays the calendar, but doesn't filter out the event. Can you help me out?

**Comment by eouia** [comment]
What it means `recurring event`? Your code seems to try some specific event removal, but I guess that is not your purpose.
Describe more details about your purpose with examples.

**Reply by the-joey-o** [reply]
I have an event called "Weight Loss Phase" on my Google calendar. It lasts for several weeks. Once it's done, there's a several week break before it starts up again. He's an example: ![Screenshot_20240925-142246.png](https://github.com/user-attachments/assets/c9445b9c-eee4-437a-96d7-c84a6ace7b9d)

How do I filter out this event?

**Reply by eouia** [reply]
Do you want to hide or remove the event with that name in the calendar view?

**Comment by eouia** [comment]
```javascript
eventFilter: (event) => { return event.title !== "Weight Loss Phase" }
```

- Without filter (Original)
![image](https://github.com/user-attachments/assets/ebbc3d33-8905-4d39-9bc2-2db463f67889)

- With filter
![image](https://github.com/user-attachments/assets/326359f4-4e2b-4b18-8292-c8d415f92bc1)

**Reply by smarath3** [reply]
Yeah adding eventFilter: (event) => { return event.title !== "<whatever text you want to remove/filter out from displaying>" } in config.js works for me. Thanks.
